spec/predicates/runtime-trace.md

```diff
@@ -105,40 +106,37 @@ Identifies the specific monitor instance used to trace the runtime.
 
 URI indicating the monitor’s identity.
 
-`process` _object_, _required_
-
-`process.HostID` _string (URI)_, _required_
+`monitor.configSource` _object_, _optional_
 
-URI indicating the process host’s identity. Ex: GitLab Runner ID
+Effectively a pointer to the monitor's configuration.
 
-`process.type` _string (TypeURI)_, _required_
+`monitor.tracePolicy` _object_, _optional_
 
-URI indicating the type of process performed.
+Indicates the trace policy used for the monitoring event that generated the
+runtime trace.
+FIXME: should this be optional?
```

Comment:
The example uses an object with one field which is a list. I'll update the description to say it's monitor specific and parsing it is based on the monitor ID.

Also, I added the fixme in the latest iteration because I think we need to discuss making the field required.